Sources/NSUI/Image.swift

```diff
@@ -16,6 +16,18 @@ extension Image {
     }
 }
 
+@available(iOS 16.0, macOS 13.0, *)
```

Comment:
tvOS missing here. Super easy to miss.

Reply:
Fixed. added watchOS too for shits and giggles

Reply:
😃